modules/ROOT/pages/provisioning-azure.adoc

```diff
@@ -21,8 +21,8 @@ Once you have picked the relevant stream, download, verify, and decompress the l
 
 [source, bash]
 ----
-stream="stable"
-coreos-installer download --decompress -s "${stream}" -p azure -f vhd.xz
+STREAM="stable"
```

Comment:
Mixed feeling here. Shouldn't we instead remove all the brackets & quotes for all of those invocations as they will never need them?

Reply:
Yeah, that's a good point – I'll get that clean up.